src/script/page/RightSidebar/ConversationDetails/utils/getConversationActions.ts

```diff
@@ -163,6 +163,15 @@ const getConversationActions = ({
         label: t('conversationDetailsActionDelete'),
       },
     },
+    {
+      condition: conversationEntity.isGroup() && conversationEntity.isSelfUserRemoved(),
+      item: {
+        click: async () => actionsViewModel.removeConversation(conversationEntity),
```

Comment:
After removing promise wrapper:
```suggestion
        click: () => actionsViewModel.removeConversation(conversationEntity),
```